drop-storage/src/lib.rs

```diff
@@ -293,16 +293,18 @@ impl Storage {
         &self,
         transfer_id: Uuid,
         path_id: String,
+        base_dir: String,
```

Comment:
I'm not sure why we take the owned string in the DB methods, `&str` should suffice